src/main/kotlin/cacophony/controlflow/functions/GenerateCall.kt

```diff
@@ -66,10 +72,15 @@ fun generateCall(
     }
 
     nodes.add(CFGNode.Call(function))
-    nodes.add(CFGNode.AdditionAssignment(rsp, CFGNode.ConstantLazy { alignmentShift.value + 8 * stackArguments.size }))
+    nodes.add(CFGNode.AdditionAssignment(rsp, CFGNode.ConstantLazy { alignmentShift.value + stackShift }))
```

Comment:
I'm not sure if we want to use stackshift here, I think this addition was to remove all arguments from stack, but we still need access to values returned on stack - it depends on the rest of implementation